README.md

```diff
@@ -143,6 +143,24 @@ Also this data doesn't include the **STATE** information, which sometimes can be
 
 Feel free to use the library if you want to automate the data access and analyze the german climate. Be aware that it could happen that the server is blocking the ftp client once in a while. It could be useful though to use a try-except-block and retry to get the data. For further examples of this library check the notebook **python_dwd_example.ipynb** in the **example** folder!
 
+## 8. Getting started
+
+```
+# Acquire sources
+git clone https://github.com/earthobservations/python_dwd
+cd python_dwd
+
+# Install dependencies
+poetry install
```

Comment:
Just for clarification: `poetry` is your defined entrypoint to the installation script? And the installation is managed by nox which is also new to me. But it looks good to me.

Reply:
You should still be able to call **pip install** via the **pyproject.toml**

Reply:
Hi there,

all of this is also pretty much new to me and I understand your feelings. However, I just wanted to give this a shot in order to implement what I learned from following the state of the onion in Python packaging through the last years. When looking at the new PEPs and at Poetry the other day, I also got weird feelings because I totally got used to the well known `setup.py`/`requirements.txt` metadata files. However, I got the understanding that this is the future of Python packaging and project management so we might want to give this a try. As I wanted to learn more about it, I just went ahead on this.

With kind regards,
Andreas.

Reply:
I love surfing on the future train. For me it is important to understand the benefits of new implementations. Otherwise I would say: "Never change a running team"